.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,76 @@
+<!-- Type of change
+Please label this PR with one of the following labels, depending on the scope of your change:
+- Bug
+- Enhancement
+- Breaking change
+-->
+
+## What does this PR do?
+
+<!-- Mandatory
+Explain here the changes you made on the PR. Please explain the WHAT: patterns used, algorithms implemented, design architecture, message processing, etc.
+-->
+
+## Why is it important?
+
+<!-- Mandatory
+Explain here the WHY, or the rationale/motivation for the changes.
+-->
+
+## Checklist
+
+<!-- Mandatory
+Add a checklist of things that are required to be reviewed in order to have the PR approved
+
+List here all the items you have verified BEFORE sending this PR. Please DO NOT remove any item, striking through those that do not apply.
+-->
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
```

Comment:
How about adding `- [ ] I have made corresponding change to the default configuration files`.